packages/mui-material/src/Alert/Alert.d.ts

```diff
@@ -4,13 +4,18 @@ import { SxProps } from '@mui/system';
 import { IconButtonProps, InternalStandardProps as StandardProps, SvgIconProps, Theme } from '..';
 import { PaperProps } from '../Paper';
 import { AlertClasses } from './alertClasses';
+import { SlotProps } from '../utils/types';
 
 export type AlertColor = 'success' | 'info' | 'warning' | 'error';
 
 export interface AlertPropsVariantOverrides {}
 
 export interface AlertPropsColorOverrides {}
 
+export interface AlertCloseButtonSlotPropsOverrides {}
+
+export interface AlertCloseIconSlotPropsOverrides {}
```

Comment:
Do we need to create these interfaces?

Reply:
It's used if users want to extend the respective slot's props with custom props. Do you think we shouldn't create them?

Reply:
We can discuss it separately but I would not add it in deprecation PRs (At least, I haven't come across a use case where I have to extend these types before). cc @mnajdova

Reply:
Agree, let's leave it for later. For now, let's just deprecate and use the same types as before.

Reply:
Done 😃